Tests/CryptoTests/Signatures/ECDSA/ECDSASignatureTests.swift

```diff
@@ -453,6 +453,138 @@ class SignatureTests: XCTestCase {
         let compressedX963Positive = Data(base64Encoded: "A+QHCXtGd5WWSQgp37FBPXMy+nnSwFK79QQD0ZeNMv7L")!
         XCTAssertThrowsError(try P256.Signing.PublicKey(x963Representation: compressedX963Positive))
     }
-    
+	
+	func testP256SigningPublicKeyEquatable() throws {
```

Comment:
Equatable, Part of #174 but I was unable to create a diff between just this PR and the source branch of that PR.